fal/controllers/anime_stats.py

```diff
@@ -40,15 +53,18 @@ def get_forum_posts(anime: Anime) -> int:
 
     if len(episode_discussions) < week:
         print(f"""
-            WARNING: did not find as many episode discussion threads for \
-                {anime.name} as the number of weeks we're in. Double check that \
+            WARNING: did not find as many episode discussion threads for 
+                {anime.name} as the number of weeks we're in. Double check that
                     this is expected and manually update if necessary.
             """)
 
+        if not episode_discussions:
```

Comment:
This if statement seems redundant because the generator expression will return 0 if episode_discussions is an empty list.

Reply:
Yeah.... I added this because it didn't seem to work without it, but I see what you're saying. Lemme try again and if I can properly see if/why it's not redundant